ArcGISRuntimeSDKQt_QMLSamples/Analysis/FindServiceAreasForMultipleFacilities/FindServiceAreasForMultipleFacilities.qml

```diff
@@ -0,0 +1,147 @@
+                    if(loadStatus === Enums.LoadStatusLoaded) {
+                        mapView.setViewpointGeometryAndPadding(fullExtent, 100);
+                    }
+                }
+                onErrorChanged: {
+                    console.warn(error.message); }
+
+                renderer: SimpleRenderer {
+                    PictureMarkerSymbol {
+                        url: imageUrl
+                        height: 25
+                        width: 25
+                    }
+                }
+            }
+
+            ServiceAreaTask {
+                id: serviceAreaTask
+                url: serviceTaskAreaUrl
+
+                onErrorChanged: {
+                    console.warn(error.message); }
+                onLoadStatusChanged: {
+                    if (loadStatus === Enums.LoadStatusLoaded) {
+                        createDefaultParameters();
+                    }
+                }
+
+                onCreateDefaultParametersStatusChanged: {
+                    if (createDefaultParametersStatus === Enums.TaskStatusCompleted) {
+                        createDefaultParametersResult.polygonDetail = Enums.ServiceAreaPolygonDetailHigh;
+                        createDefaultParametersResult.returnPolygons = true;
+
+                        // add service areas of 1 minute and 3 minutes travel time by car
+                        createDefaultParametersResult.defaultImpedanceCutoffs = [1.0, 3.0];
+                        createDefaultParametersResult.setFacilitiesWithFeatureTable(featureTable, queryParameters);
+
+                        solveServiceArea(createDefaultParametersResult);
+                    }
+                }
+
+                onSolveServiceAreaStatusChanged: {
+                    if (solveServiceAreaStatus === Enums.TaskStatusCompleted) {
+                        // iterate through the facilities to get the service area polygons
+                        for (var i = 0; i < solveServiceAreaResult.facilities.length; i++) {
+                            var serviceAreaPolygonList = solveServiceAreaResult.resultPolygons(i);
+                            //create a graphic for each available polygon
+                            for (var j = 0; j < serviceAreaPolygonList.length; j++) {
+                                serviceAreasOverlay.graphics.append(ArcGISRuntimeEnvironment.createObject("Graphic", {geometry: serviceAreaPolygonList[j].geometry, symbol: fillSymbols[j]}));
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        RowLayout {
+            anchors {
+                left: parent.left
+                top: parent.top
+            }
+
+            Button {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                text: "Find Service Areas"
+                onClicked: {
+                    enabled = false;
+                    serviceAreaTask.load();
```

Comment:
I think that the load should happen on the component complete for the task and when you cllick find service areas it just does the solve part